oxidation/libparsec/crates/core/src/events.rs

```diff
@@ -0,0 +1,87 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+#[derive(Clone, PartialEq, Eq, Debug)]
```

Comment:
Can you put `Debug` first ? I like to have essential derive come first

Reply:
Done.